include/material/mohr_coulomb.tcc

```diff
@@ -432,9 +440,6 @@ void mpm::MohrCoulomb<Tdim>::compute_df_dp(
   double df_dc = -1.;
   (*softening) =
       (-1.) * ((df_dphi * dphi_dpstrain) + (df_dc * dc_dpstrain)) * dp_dj;
-
-  // Check the epds
-  if (epds < epds_peak_ && epds > epds_peak_) (*softening) = 0;
```

Comment:
Don't we need this check, because we no longer check it later?

Reply:
I think we we don't need this check, the *softening will equal to 0 if the dphi_dpstrain and dc_dpstrain are 0.

Reply:
Yes, we don't need this additional check.